src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_LocalDeclaration.cs

```diff
@@ -66,16 +66,18 @@ private BoundStatement RewriteLocalDeclaration(CSharpSyntaxNode syntax, LocalSym
                     localSymbol.RefKind),
                 hasErrors);
 
-            return AddLocalDeclarationSequencePointIfNecessary(syntax, localSymbol, rewrittenLocalDeclaration, wasCompilerGenerated);
+            return AddLocalDeclarationSequencePointIfNecessary(originalOpt, localSymbol, rewrittenLocalDeclaration);
         }
 
-        private BoundStatement AddLocalDeclarationSequencePointIfNecessary(CSharpSyntaxNode syntax, LocalSymbol localSymbol, BoundStatement rewrittenLocalDeclaration, bool wasCompilerGenerated = false)
+        private BoundStatement AddLocalDeclarationSequencePointIfNecessary(BoundLocalDeclaration originalOpt, LocalSymbol localSymbol, BoundStatement rewrittenLocalDeclaration)
         {
             // Add sequence points, if necessary.
```

Comment:
I'm assuming you have planned this for later update, but any mention of sequence point should be removed from local rewriter and replaced with mentions of instrumenter.